integr8ly/application-monitoring-operator
# pass namespace labels to child operators

Adds a property to the CR that allows passing down the namespace selector labels to the child operators (Grafana and Prometheus operators).

Verification steps:
1. Update `deploy/examples/ApplicationMonitoring.yaml` and put some arbitrary value into `namespaceSelector`
1. Run the operator and deploy the monitoring stack: `oc create -f deploy/examples/ApplicationMonitoring.yaml`
1. After everything is running check the Grafana and Prometheus resources: they should have your arbitrary value in their namespace selectors.
